sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/ResourceDeleteCancelEventData.java

```diff
@@ -5,14 +5,24 @@
 package com.azure.messaging.eventgrid.systemevents;
 
 import com.azure.core.annotation.Fluent;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.core.util.serializer.JacksonAdapter;
+import com.azure.core.util.serializer.SerializerAdapter;
+import com.azure.core.util.serializer.SerializerEncoding;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import java.io.IOException;
+import java.util.Map;
+
 /**
  * Schema of the Data property of an EventGridEvent for a Microsoft.Resources.ResourceDeleteCancel event. This is raised
  * when a resource delete operation is canceled.
  */
 @Fluent
 public final class ResourceDeleteCancelEventData {
+    static final ClientLogger LOGGER = new ClientLogger(ResourceDeleteCancelEventData.class);
+    final SerializerAdapter defaultSerializerAdapter = JacksonAdapter.createDefaultSerializerAdapter();
```

Comment:
This should be static too. We don't want to instantiate the serializer for each new event data instance as it's an expensive operation.